proto/penumbra/penumbra/view/v1alpha1/view.proto

```diff
@@ -418,15 +418,23 @@ message TransactionInfoByHashRequest {
 }
 
 message TransactionInfoRequest {
+  // Submessage to represent optionality for specifying heights.
+  message BlockHeight {
+      uint64 height = 1;
+  }
   // If present, return only transactions after this height.
-  optional uint64 start_height = 1;
+  BlockHeight start_height = 1;
   // If present, return only transactions before this height.
-  optional uint64 end_height = 2;
+  BlockHeight end_height = 2;
```

Comment:
Rather than defining a submessage, these should just be `uint64 start_height` and `uint64 end_height`, since 0 is not a valid block height.